.changes/fix-non-standard-permission-bug.md

```diff
@@ -0,0 +1,6 @@
+---
+'tauri': 'patch:bug'
```

Comment:
```suggestion
'tauri-bundler': 'patch:bug'
```

package name is wrong, but good one!

Reply:
I was actually confused a bit over there :sweat_smile: